### PR TITLE
Remove dependency for logic manager, update clisyntax and docs

### LINKED ISSUE #109: Update documentation for addTaskCommand

### LINKED ISSUE #110: Update documentation for doneCommand
When entered an index <= 0, and index >0 and not valid.

### LINKED ISSUE #111: Update documentation for addEventCommand

### LINKED ISSUE #117: Task and Event: Duplicates allowed

### PR DESCRIPTION
**Pull request by litone01**
Removed dependency of AB3 for logic manager.

Updated clisyntax for startdate and enddate.

Updated documentation: closes #109 , closes #110 , closes #111 

Solved bugs in identifying duplicate tasks and events (based on name only): closes #117

### CODE REVIEW

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 1]
```diff
@@ -57,16 +55,6 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
...
-        return model.getAddressBook();
-    }
-
-    @Override
-    public ObservableList<Person> getFilteredPersonList() {
-        return model.getFilteredPersonList();
-    }
-
     @Override
     public Path getAddressBookFilePath() {
```
Comment (YuFeng0930) [inline src/main/java/seedu/address/logic/LogicManager.java:59]: Should this be removed or it's waiting for model dependencies removal?

**Review by YuFeng0930 — APPROVED** [review]
LGTM, but just one thing to be confirmed.

**Review by ljhgabe — APPROVED** [review]
Except from the issue raised by Yu Feng, all else LGTM

File: src/main/java/seedu/address/logic/LogicManager.java (hunk 1 above)
Comment (litone01) [reply src/main/java/seedu/address/logic/LogicManager.java:59]: oh if you are removing it I can add it back

**Pull request merged**